nixos/modules/services/backup/restic.nix

```diff
@@ -152,6 +152,7 @@ in
           example = {
             OnCalendar = "00:05";
             RandomizedDelaySec = "5h";
+            Persistent = true;
```

Comment:
What about setting it by default?

Reply:
Okay pushed the change that makes the timer unit persistent by default, also renamed the commit :D